src/Json/Json_Parser.cpp

```diff
@@ -543,104 +543,102 @@ struct Container {
+                                       .arg(begin, QString::fromUtf8(data.constData(), data.size())));
+            }
+            break;
         }
-        if (UNLIKELY(!ok)) {
-            // this should never happen
-            throw Json::ParseError(QString("Failed to parse number from string: %1 (original: %2)")
-                                   .arg(begin, QString::fromUtf8(data.constData(), data.size())));
+        case BoolTrue:
+            ret = true;
+            break;
+        case BoolFalse:
+            ret = false;
+            break;
+        case Str:
+            // NB: data may be a shallow or deep copy of the original data in `bytes`
+            // We use this C string syntax because it's faster, as well as more correct.
+            // Also note that round1.json test fails (because it contains the 0 codepoint)
+            // unless we do this C-string syntax to construct the QString.
+            // QString quirks, see:  https://github.com/qt/qtbase/blob/ba3b53cb501a77144aa6259e48a8e0edc3d1481d/src/corelib/text/qstring.h#L701
+            //              versus:  https://github.com/qt/qtbase/blob/ba3b53cb501a77144aa6259e48a8e0edc3d1481d/src/corelib/text/qstring.h#L709
+            ret = QString::fromUtf8(data.constData(), data.size());
+            break;
+        case Arr: {
+            QVariantList vl;
+            vl.reserve(values.size());
+            for (const auto & cont : values) {
+                vl.push_back(cont.toVariant());
+            }
+            ret = vl;
+            break;
         }
-        break;
-    }
-    case BoolTrue:
-        ret = true;
-        break;
-    case BoolFalse:
-        ret = false;
-        break;
-    case Str:
-        // NB: data may be a shallow or deep copy of the original data in `bytes`
-        // We use this C string syntax because it's faster, as well as more correct.
-        // Also note that round1.json test fails (because it contains the 0 codepoint)
-        // unless we do this C-string syntax to construct the QString.
-        // QString quirks, see:  https://github.com/qt/qtbase/blob/ba3b53cb501a77144aa6259e48a8e0edc3d1481d/src/corelib/text/qstring.h#L701
-        //              versus:  https://github.com/qt/qtbase/blob/ba3b53cb501a77144aa6259e48a8e0edc3d1481d/src/corelib/text/qstring.h#L709
-        ret = QString::fromUtf8(data.constData(), data.size());
-        break;
-    case Arr: {
-        QVariantList vl;
-        vl.reserve(values.size());
-        for (const auto & cont : values) {
-            vl.push_back(cont.toVariant());
+        case Obj: {
+            // NB: pair.first in entries may be a deep or shallow copy of the data in `bytes`
+            QVariantMap vm;
+            for (const auto & [key, cont] : entries) {
+                // We use this C string syntax because it's faster & more accurate. (QString quirks)
+                vm[QString::fromUtf8(key.constData(), key.size())] = cont->toVariant();
```

Comment:
All you changed was this 1 line yet the diff is huge because you indented this whole function 1 level... lol.

Reply:
It is indented one level because it is now inside the claudators above with the `struct Container`

Reply:
> claudators

What is this word?

Reply:
Oh I see you made it inline in the class def. Doesn't need to live there. Makes the diff too big...